### PR TITLE
zhanyi/updatek8sreadme

### PR DESCRIPTION
**Pull request by mszhanyi**
update k8s readme to add Azure AKS creation.

### CODE REVIEW

File: kubernetes/README.md [hunk 1]
```diff
@@ -3,35 +3,36 @@
...
+are not using AWS or Azure, please refer to your cloud/infrastructure provider's manual
+to setup a kubernetes cluster.
 
-> **NOTE**: EKS is not required to run this controller, 
+> **NOTE**: EKS/AKS is not required to run this controller,
 >  you can use other Kubernetes clusters.
 
-Use `eksctl` to create an Amazon EKS cluster. This process takes ~15 minutes. 
+#### Create your cluster
+1. Use `eksctl` to create an Amazon EKS cluster. This process takes ~15 minutes.
```
Comment (nbcsm) [inline kubernetes/README.md]: nit: how about something like this? 
`1. 2. `feels a little confusing here.

```
### (Optional) Setup
#### Create your cluster
##### Create EKS
##### Create AKS
#### Install Nvidia device plugin to enable GPU support on your cluster.
```

File: kubernetes/README.md [hunk 2]
```diff
@@ -44,22 +45,35 @@ eksctl create cluster \
...
+    --location westus2 \
+    --kubernetes-version 1.15
+    --generate-ssh-keys
+```
+
+#### Install Nvidia device plugin to enable GPU support on your cluster.
 Deploy the following Daemonset:
 
 ```shell
 kubectl create -f https://raw.githubusercontent.com/NVIDIA/k8s-device-plugin/1.0.0-beta4/nvidia-device-plugin.yml
```
Comment (nbcsm) [inline kubernetes/README.md:67]: to double check, is this necessary for AKS?

Comment (mszhanyi) [reply kubernetes/README.md:67]: Yes, It's necessary for all K8s with Nvidia GPU

File: kubernetes/README.md (hunk 1 above)
Comment (mszhanyi) [reply kubernetes/README.md]: updated

**Pull request merged**